deployment/refinery-modules/refinery/manifests/aws.pp

```diff
@@ -69,7 +69,13 @@
   group => "$app_user",
   mode => "0755",
 }
-
+->
+file { $::django_docker_engine_data_dir:
```

Comment:
This creates `$django_docker_engine_data_dir` on AWS only. What about on Vagrant?

Reply:
I don't think we care so much on Vagrant because a temporary directory will be created... @mccalluc Is this true?

Reply:
see the discussion on default.pp below.